README.md

```diff
@@ -80,13 +81,80 @@ In the end the PCB will not be visible from the outside, powered from the inside
 ![](images/PCBv1.7.png)
 
 For now, please follow the details given in the schematics and the pictures located in the [images](images) folder.
+
+## Openhab-Config Example
```

Comment:
I think this would be more correct:
> ## openHAB Configuration Example

Also just for the looks of it, could you please add a filler sentence? Something like:

> The following openHAB configuration allows integration of the Senseo machine with the home automation solution openHAB. Please make sure the machine is connected to your MQTT broker. Insert the MQTT broker IP and replace "senseo-wifi-home" by your device name in the below example.